pkg/interface/publish/src/js/components/lib/note.js

```diff
@@ -165,7 +165,7 @@ export class Note extends Component {
               popout={props.popout}
               sidebarShown={props.sidebarShown}
             />
-            <Link className="f9 w-100 w-90-m w-90-l mw6 tl" to={baseUrl}>
+            <Link className="f9 w-50 w-100-m w-100-s center tl" to={baseUrl}>
```

Comment:
Careful — this regresses an issue the previous weird class stuff fixed on `-m` viewports:

<img width="394" alt="image" src="https://user-images.githubusercontent.com/20846414/76360623-bf0b8680-62f3-11ea-9560-d6c383e6fbf8.png">